Physical_Engine/Query/HasMergeablePropertiesWith.cs

```diff
@@ -47,6 +47,9 @@ public static partial class Query
         [Output("equal", "True if the IFramingElements' non-geometrical property is equal to the point that they could be merged into one object.")]
         public static bool HasMergeablePropertiesWith(this IFramingElement element, IFramingElement other)
         {
+            if(element == null || other == null)
+                return false; //If either is null, then it can probably can't have its properties merged
```

Comment:
Shouldn't there be a warning raised here? Refers to the code change below as well.

Reply:
This was to keep it consistent with other engines which had not opted to raise a warning on this particular method if `null`s were present.